src/sentry/workflow_engine/endpoints/project_detector_index.py

```diff
@@ -79,12 +85,23 @@ def get(self, request, project):
         """
         queryset = Detector.objects.filter(
             project_id=project.id,
-        ).order_by("id")
+        )
+
+        sort_by = request.GET.get("sortBy", "id")
+        sort_field = sort_by[1:] if sort_by.startswith("-") else sort_by
+        if sort_field not in SORT_ATTRS:
+            sort_field = "id"
```

Comment:
I _think_ it makes sense to default to `id`, but we do return a 400 in some spots instead 🤔